src/Agenda.js

```diff
@@ -70,7 +70,7 @@ class Agenda extends React.Component {
                 </tbody>
               </table>
             </div>
-          </React.Fragment>
+          </div>
```

Comment:
this isn't a free change, it adds markup that wasn't there before, which i'm not super comfortable with since that likely will break ppl's apps and cause unclear regressions